src/main/java/collections/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,36 @@
+package collections.coffee.order;
+
+import java.util.TreeMap;
+
+public class CoffeeOrderBoard {
+    final TreeMap<Integer, String> orders = new TreeMap<>();
+
+    public void add(int orderNumber, String name) throws DuplicateNumberException {
+        if (!orders.containsKey(orderNumber)) {
+            orders.put(orderNumber, name);
+        } else {
+            throw new DuplicateNumberException(orderNumber);
+        }
+    }
+
+    public void deliver() {
+        orders.remove(orders.firstKey());
+    }
+
+    public void deliver(int orderToDelete) throws WrongNumberToDeliver {
+        if (orders.containsKey(orderToDelete)) {
+            orders.remove(orderToDelete);
+        } else {
+            throw new WrongNumberToDeliver(orderToDelete);
+        }
+    }
+
+    public void draw(CoffeeOrderBoard board) {
```

Comment:
```suggestion
    public List<Order> draw(CoffeeOrderBoard board) {
```
Інакше не протестуєш.